taxcalc/tests/test_parameters.py

```diff
@@ -116,23 +149,24 @@ def update_params(self, revision,
         assert prms.wage_growth_rates() == list()
         prms.set_year(2010)
         assert prms.current_year == 2010
-        with pytest.raises(ValueError):
+        with pytest.raises(paramtools.ValidationError):
             prms.set_year(2011)
         return
+
     if expect == 'raise':
-        with pytest.raises(ValueError):
-            prms.update_params(revision,
-                               print_warnings=False,
-                               raise_errors=False)
+        with pytest.raises(paramtools.ValidationError):
+            prms.update_params(revision)
     elif expect == 'noerror':
-        prms.update_params(revision, print_warnings=False, raise_errors=False)
-        assert not prms.parameter_errors
+        prms.update_params(revision)
+        assert not prms.errors
     elif expect == 'error':
-        prms.update_params(revision, print_warnings=False, raise_errors=False)
-        assert prms.parameter_errors
+        with pytest.raises(paramtools.ValidationError):
+            prms.update_params(revision)
+        assert prms.errors
     elif expect == 'warn':
-        prms.update_params(revision, print_warnings=False, raise_errors=False)
-        assert prms.parameter_warnings
+        with pytest.raises(paramtools.ValidationError):
+            prms.update_params(revision)
+        assert prms.warnings
```

Comment:
@Peter-Metz Changes in this section are related to differences in error handling:
- `print_warning` vs `ignore_warning`
- run warning level validators vs not running them